decidim-core/app/cells/decidim/progress_bar/show.erb

```diff
@@ -7,7 +7,7 @@
   <%= content_tag :div, units_name_text, class: "progress-bar__units" if units_name %>
 
   <% if total != 0 %>
-    <div class="progress-bar" role="progressbar" tabindex="0" aria-valuenow="<%= percentage %>" aria-valuemin="0" aria-valuetext="<%= percentage %> percent" aria-valuemax="<%= total %>">
+    <div class="progress-bar" role="progressbar" tabindex="0" aria-label="<%= percentage %>" aria-valuenow="<%= percentage %>" aria-valuemin="0" aria-valuetext="<%= percentage %> percent" aria-valuemax="<%= total %>">
```

Comment:
This is incorrect for the `aria-label`. It should describe the purpose of the progress bar, not the current percentage value which is already described using `aria-valuetext`.

See:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/progressbar_role

>[aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) or [aria-labelledby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby)
> Defines the string value or identifies the element (or elements) that label the progressbar element providing an accessible name. An accessible name is required.

Reply:
Done